www-apps/davical/davical-1.1.7-r2.ebuild

```diff
@@ -1,27 +1,28 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit webapp
 
 DESCRIPTION="A CalDAV and CardDAV Server"
-HOMEPAGE="https://davical.org/"
+HOMEPAGE="https://www.davical.org/"
 SRC_URI="https://www.davical.org/downloads/${PN}_${PV}.orig.tar.xz -> ${P}.tar.xz"
 
-LICENSE="GPL-2+"
+LICENSE="GPL-2 GPL-2+ GPL-3+ LGPL-2.1+ LGPL-3+"
 KEYWORDS="~amd64 ~x86"
+IUSE="ldap"
 
-DEPEND=">=dev-php/awl-0.59
-	sys-devel/gettext"
 RDEPEND="app-admin/pwgen
-	dev-lang/php:*[calendar,curl,pdo,postgres,xml]
+	dev-lang/php:*[calendar,curl,iconv,imap,ldap?,nls,pdo,postgres,xml]
 	dev-perl/DBD-Pg
 	dev-perl/DBI
 	dev-perl/YAML
 	>=dev-php/awl-0.59
 	virtual/httpd-php"
 
+DEPEND="sys-devel/gettext"
```

Comment:
This is most likely a BDEPEND with EAPI-7.

Reply:
you are right, I was to fast simply re-adding it as present before removal